justfile

```diff
@@ -189,6 +189,8 @@ create platform=default_platform:
             :
         ;;
         "AKS-PEER-SNP")
+            just upload-image
```

Comment:
This should be decoupled from create so we don't need to recreate the cluster to iterate on the image.

Reply:
How about we also add the `just upload-image` and `terraform apply` steps right before the `nix run -L .#scripts.deploy-caa` step in node-installer?

We need this step here because Terraform won't work with the image_id variable missing.
Creating the `just.auto.tfvars` config is idempotent and `terraform apply` is as well. I don't know what `terraform init` is doing.

Reply:
Depends on how well we can detect whether we need to upload or not.